src/main/java/ljv/LJV.java

```diff
@@ -24,253 +24,230 @@
 import java.util.*;
 
 class LJV {
-    private final IdentityHashMap<Object, String> objectsId = new IdentityHashMap<>();
+    private final Map<Object, String> classAttributeMap = new HashMap<>();
+    private final Map<Object, String> fieldAttributeMap = new HashMap<>();
+    private final Set<Object> pretendPrimitiveSet = new HashSet<>();
+    private final Set<Object> ignoreSet = new HashSet<>();
+
+    private Direction direction;
+
+    public LJV setDirection(Direction direction) {
+        this.direction = direction;
+        return this;
+    }
+    public String getDirection() {
```

Comment:
OMG, no need to do this.

First, "LR" etc is Grafviz's words, and LJV class should know nothing about Grafviz.

Second, you use series of `if..`, while there are better alternatives: `switch`, or even better alternative -- to define some getter in Enum itself (see [Lecture 3](https://inponomarev.github.io/java-mipt/slides04/#/_%D0%BF%D0%BE%D0%BB%D1%8F_%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B_%D0%B8_%D0%BA%D0%BE%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D0%BE%D1%80%D1%8B_%D0%B4%D0%BB%D1%8F_%D0%BF%D0%B5%D1%80%D0%B5%D1%87%D0%B8%D1%81%D0%BB%D0%B5%D0%BD%D0%B8%D0%B9))

But in this particular case it can be even simpler. You may return direction itself, in order to convert it to Grafviz's direction you can use `.toString()` call in `GraphBuilder` class